examples/bri-3/components/storage/sql-connection.js

```diff
@@ -0,0 +1,79 @@
+const { Client } = require('pg');
+
+//Terminal cmd to spin up pg docker image
+//docker run --name postgres -e POSTGRES_PASSWORD=example -p 5433:5432 -d postgres 
+
+//Local DB
+// const client = new Client({
+//     user: 'postgres',
+//     host: 'localhost',
+//     database: 'postgres',
+//     password: '',
+//     port: 5432,
+// });
+
+//Docker DB
+const client = new Client({
```

Comment:
multiple db environments should be in multiple env files, not sure what is storate/database.json used for?